editor/action_map_editor.cpp

```diff
@@ -573,6 +575,11 @@ ActionMapEditor::ActionMapEditor() {
 	show_builtin_actions_checkbutton->connect("toggled", callable_mp(this, &ActionMapEditor::set_show_builtin_actions));
 	add_hbox->add_child(show_builtin_actions_checkbutton);
 
+	show_builtin_actions = EditorSettings::get_singleton()->get_project_metadata("project_settings", "show_builtin_actions", false);
+	if (show_builtin_actions) {
+		show_builtin_actions_checkbutton->set_pressed(true);
```

Comment:
I'd remove the above assignment of false (line 573) and make it a single `set_pressed` with the loaded value

Reply:
Sounds great !